core/src/main/scala/cats/Alternative.scala

```diff
@@ -2,5 +2,20 @@ package cats
 
 import simulacrum.typeclass
 
-@typeclass trait Alternative[F[_]] extends Applicative[F] with MonoidK[F]
+@typeclass trait Alternative[F[_]] extends Applicative[F] with MonoidK[F] { self =>
+  /**
+   * Compose two Alternative intsances.
+   */
+  def compose[G[_]](implicit GG: Alternative[G]): Alternative[λ[α => F[G[α]]]] =
```

Comment:
I think the `GG` constraint can be relaxed to `Applicative` while still returning an `Alternative` (also on `CompositeAlternative` below).